util/mdbook-reggen

```diff
@@ -19,16 +19,11 @@ from reggen.ip_block import IpBlock
 import reggen.gen_cfg_html as gen_cfg_html
```

Comment:
Is there any reason why this commit cannot be squashed into the previous reggen commit?
https://github.com/lowRISC/opentitan/pull/17323/commits/0a1340921d27990c5dc8291434b3ccc307db044f